duckduckgo/duckduckgo-privacy-extension
# Expand breakage params

**Reviewer:** @sammacbeth 



## Description:

https://app.asana.com/0/1163321984198618/1207084200685811/f

This PR add 4 new parameters to breakage reports: userRefreshCount, locale, jsPerformance, and openerContext

## Steps to test this PR:
1. Open a page
2. Refresh a few times
3. Submit a breakage report
4. Inspect the call to ensure the new parameters are included

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
